RecoEgamma/EgammaPhotonProducers/interface/ReducedEGProducer.h

```diff
@@ -90,18 +94,23 @@ class ReducedEGProducer : public edm::stream::EDProducer<> {
  const std::string outESRecHits_;
  const std::string outPhotonPfCandMap_;
  const std::string outGsfElectronPfCandMap_;
+ const std::string outGsfTracks_;  //add by me - Hien xinh dep
+ //const std::string outGsfTrackCores_; //add by me - Hien xinh dep
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

if you need to keep these commants, please cleanup to be clear who "me" is ; I think this is not needed at all